content-src/components/Snippet/Snippet.js

```diff
@@ -0,0 +1,37 @@
+const React = require("react");
+const classNames = require("classnames");
+
+const Snippet = React.createClass({
+  onClose(e) {
+    e.preventDefault();
+    this.props.setVisibility(false);
+  },
+  render() {
+    const {visible, description, title, image} = this.props;
+    return (<div className={classNames("snippet", {"hidden": !visible})}>
+      <div className="snippet-container">
+        <div ref="image" className={classNames("snippet-image", {"placeholder": !image})} style={{backgroundImage: image ? `url(${image})` : "none"}} />
+        <div className="snippet-text">
+          <h3 ref="title" hidden={!title} className="snippet-title">{title}</h3>
```

Comment:
Another place where we can simply not render if title isn't set, instead of rendering and hiding.